jms-core/src/main/java/io/micronaut/jms/configuration/JMSQueueListenerMethodProcessor.java

```diff
@@ -58,21 +58,24 @@ protected ExecutorService getExecutorService(AnnotationValue<Queue> value) {
         final Optional<String> executorName = value.stringValue("executor");
         final Optional<String> concurrency = value.stringValue("concurrency");
 
+        if (!executorName.isPresent() && !concurrency.isPresent()) {
+            return null;
+        }
+
         if (executorName.isPresent() && !executorName.get().isEmpty()) {
             return beanContext.findBean(ExecutorService.class, Qualifiers.byName(executorName.get()))
                 .orElseThrow(() -> new IllegalStateException(
                     "No ExecutorService bean found with name " + executorName.get()));
         }
```

Comment:
why do we still need this code if the members were deprecated?

Reply:
it is to allow backward compatibility for the usage of the annotation concurency options before we completely remove concurrency support. If the annotation concurency options are configured, the implementation is utilized as before. If options are not present, the will not be conurency utilization at jms meesagee proccessing level. 

Do you want to say that we should remove the concurency utilization regardles of presence of the deprecated anotation concurency options?